common/src/main/java/com/wynntils/features/redirects/ChatRedirectFeature.java

```diff
@@ -133,8 +132,6 @@ private void register(Redirector redirector) {
 
     @SubscribeEvent(priority = EventPriority.HIGH)
     public void onChatMessage(ChatMessageReceivedEvent e) {
-        if (e.getRecipientType() != RecipientType.INFO) return;
```

Comment:
So why did you not change it to GUILD instead?

Reply:
Because this event applies all chat redirects. We are still in debt regarding this one.

Reply:
Ah, now I see. Previously we only redirected info messages, now we also have a guild one. 

But let me rephrase it:

So why did you not change it to INFO or GUILD instead?

That this is dropped means we will check for redirects in all global chat messages etc.

Reply:
Yes, we can do that. I've not had that idea. If I remember next time I'm developing, I'll PR it.